packages/components/src/components/segmented-controls/segmented-control/segmented-control.scss

```diff
@@ -0,0 +1,65 @@
+@use "~@infineon/design-system-tokens/dist/tokens";
```

Comment:
![image](https://github.com/user-attachments/assets/3eeb3db1-4c84-4618-84d8-3556fe44ebd1)
should they not be all the same size?